components/telemetry-operator/controller/tracepipeline/resources.go

```diff
@@ -202,6 +203,24 @@ func makeProcessorsConfig() map[string]any {
 	}
 }
 
+func makeFilterProcessorConfig() map[string]any {
+	return map[string]any{
+		"traces": map[string]any{
+			"span": []any{
+				"(attributes[\"http.method\"] == \"GET\") and (attributes[\"component\"] == \"proxy\") and (attributes[\"OperationName\"] == \"Egress\") and (resource.attributes[\"service.name\"] == \"jaeger.kyma-system\")",
+				"(attributes[\"http.method\"] == \"GET\") and (attributes[\"component\"] == \"proxy\") and (attributes[\"OperationName\"] == \"Egress\") and (resource.attributes[\"service.name\"] == \"grafana.kyma-system\")",
+				"(attributes[\"http.method\"] == \"GET\") and (attributes[\"component\"] == \"proxy\") and (attributes[\"OperationName\"] == \"Ingress\") and (IsMatch(attributes[\"http.url\"], \".+/metrics\") == true)",
+				"(attributes[\"http.method\"] == \"GET\") and (attributes[\"component\"] == \"proxy\") and (attributes[\"OperationName\"] == \"Ingress\") and (IsMatch(attributes[\"http.url\"], \".+/healthz(/.*)?\") == true)",
```

Comment:
For the metrics and helathz where we do not use the service.name for proper identification of the source but only a match expresseion, we maybe should limit the source to the kyma-system namespace by using the related resource attribute? For that you would need to move the processor behind the attribute processor in the chain